frontend/talentsearch/src/js/components/createAccount/CreateAccountPage.tsx

```diff
@@ -0,0 +1,329 @@
+                govEmployeeStatus={govEmployeeStatus}
+                groupSelection={groupSelection}
+              />
+              <div
+                data-h2-margin="b(top-bottom, l)"
+                data-h2-padding="b(top-bottom, l)"
+                data-h2-border="b(lightgray, top, solid, s)"
+                data-h2-display="b(flex)"
+                data-h2-justify-content="b(flex-end)"
+              >
+                <Submit
+                  mode="solid"
+                  color="primary"
+                  text={intl.formatMessage({
+                    defaultMessage: "Save and go to my profile",
+                    description:
+                      "Button label for submit button on create account form.",
+                  })}
+                />
+              </div>
+            </div>
+          </form>
+        </FormProvider>
+      </div>
+    </section>
+  );
+};
+
+export const CreateAccount: React.FunctionComponent = () => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const paths = talentSearchRoutes(locale);
+
+  const [lookUpResult] = useGetCreateAccountFormDataQuery();
+  const { data: lookupData, fetching: fetchingLookupData } = lookUpResult;
+  const meInfo = lookupData?.me;
+  const meId = meInfo?.id;
+  const classifications: Classification[] | [] =
+    lookupData?.classifications.filter(notEmpty) ?? [];
+  const departments: Department[] | [] =
+    lookupData?.departments.filter(notEmpty) ?? []; // TODO: Add department field to User in separate issue.
+
+  const [, executeMutation] = useCreateAccountMutation();
+  const handleCreateAccount = (id: string, data: UpdateUserAsUserInput) =>
+    executeMutation({
+      id,
+      user: data,
+    }).then((result) => {
+      if (result.data?.updateUserAsUser) {
+        return result.data.updateUserAsUser;
+      }
+      return Promise.reject(result.error);
+    });
+
+  const onSubmit = async (data: UpdateUserAsUserInput) => {
+    if (meId === undefined || meId === "") {
+      toast.error(
+        intl.formatMessage({
+          defaultMessage: "Error: user not found",
+          description: "Message displayed to user if user is not found",
+        }),
+      );
+      return;
+    }
+    await handleCreateAccount(meId, data)
+      .then(() => {
+        navigate(paths.profile());
+        toast.error(
+          intl.formatMessage({
+            defaultMessage: "Account successfully created.",
+            description:
+              "Message displayed to user if account is created successfully.",
+          }),
+        );
+      })
+      .catch(() => {
+        toast.error(
+          intl.formatMessage({
+            defaultMessage: "Error: creating account failed.",
+            description:
+              "Message displayed to user if account fails to get updated.",
+          }),
+        );
+      });
+  };
+
+  if (fetchingLookupData) {
+    return <p>{intl.formatMessage(commonMessages.loadingTitle)}</p>;
+  }
+
```

Comment:
we've got a loading component now, that may be appropriate here
in common components called `pending`